ddspipe_core/src/cpp/communication/dds/DdsBridge.cpp

```diff
@@ -175,6 +97,178 @@ void DdsBridge::disable() noexcept
     }
 }
 
+void DdsBridge::create_all_tracks()
```

Comment:
The new methods are not thread safe, are they?

Reply:
They aren't. You're right.